content/pt-br/docs/concepts/architecture/nodes.md

```diff
@@ -0,0 +1,392 @@
+
+O terceiro é monitorar a saúde dos nós. O controlador do nó é responsável por:
+
+- No caso de um nó se tornar inacessível, atualizando a condição NodeReady de dentro do `.status` do nó. Nesse caso, o controlador do nó define a condição de pronto (`NodeReady`) como condição desconhecida (`ConditionUnknown`).
+- Se um nó permanecer inacessível: será iniciado o [despejo pela API](/docs/concepts/scheduling-eviction/api-eviction/) para todos os Pods no nó inacessível. Por padrão, o controlador do nó espera 5 minutos entre marcar o nó como condição desconhecida (`ConditionUnknown`) e enviar a primeira solicitação de despejo.
+
+O controlador de nó verifica o estado de cada nó a cada `--node-monitor-period` segundos.
+
+### Limites de taxa de despejo
+
+Na maioria dos casos, o controlador de nós limita a taxa de despejo a `--node-eviction-rate` (0,1 por padrão) por segundo, o que significa que ele não despejará pods de mais de 1 nó por 10 segundos.
+
+O comportamento de despejo do nó muda quando um nó em uma determinada zona de disponibilidade se torna não saudável. O controlador de nós verifica qual porcentagem de nós na zona não são saudáveis (a condição `NodeReady` é desconhecida `ConditionUnknown` ou falsa `ConditionFalse`) ao mesmo tempo:
+
+- Se a fração de nós não saudáveis for ao menos `--unhealthy-zone-threshold` (padrão 0,55), então a taxa de despejo será reduzida.
+- Se o cluster for pequeno (ou seja, tiver menos ou igual a nós `--large-cluster-size-threshold` - padrão 50), então os despejos serão interrompidos.
+- Caso contrário, a taxa de despejo é reduzida para `--secondary-node-eviction-rate` de despejo de nós secundários (padrão 0,01) por segundo.
+
+A razão pela qual essas políticas são implementadas por zona de disponibilidade é porque uma zona de disponibilidade pode ser particionada a iniciar do plano de controle, enquanto as outras permanecem conectadas. Se o seu cluster não abranger várias zonas de disponibilidade de provedores de nuvem, o mecanismo de despejo não levará em conta a indisponibilidade por zona.
+
+Uma das principais razões para espalhar seus nós pelas zonas de disponibilidade é para que a carga de trabalho possa ser transferida para zonas saudáveis quando uma zona inteira cair. Portanto, se todos os nós em uma zona não forem saudáveis, o controlador do nó despeja na taxa normal de `--node-eviction-rate`. O caso é quando todas as zonas são completamente insalubres (nenhum dos nós do cluster será saudável). Nesse caso, o controlador do nó assume que há algum problema com a conectividade entre o plano de controle e os nós e não realiza nenhum despejo. (Se houver uma interrupção e alguns nós reaparecerem, o controlador do nó expulsa pods dos nós restantes que são insalubres ou inacessíveis).
```

Comment:
I did not find the translation for "corner case" here. Should it have been "caso especial" instead of just "caso"?